apps/base-docs/src/theme/Navbar/Content/index.js

```diff
@@ -50,6 +55,9 @@ function NavbarLayoutTopContent({ left, right }) {
             target="_blank"
             rel="noreferrer"
             aria-label="Base on Discord"
+            onClick={() => {
```

Comment:
These can all be callbacks

Reply:
Understood, I've created callbacks for each social click in the next PR.